### PR TITLE
Update to version 4.0.0

### PR DESCRIPTION
**Pull request by danielfdsilva**
Following up on our conversation in the last standardization meeting I'm pushing an updated version of project seed.
This is from one of the latest projects set up using this starter, with dependencies updated and minor tweaks.

> As we discussed this update does not mean that we've decided on using project-seed as the go-to standard (at time of writing)

# Changelog

## 🎉 Features
- Added React as a default. The vast majority of our project are React. It's easy to remove it if need be, than the other way around.
- Added Styled Components and Collecticons (1) as a default. Same reasoning as above. (2)
- Added Prettier as a dependency. It is now easy to use it with VS Code (3). Linting rules are the same we're using in the new DSo.

## 🚀 Improvements
- Node 14 as default
- Update Circle CI config to use workflows
- Update all dependencies to latest version
- Add metatags to index to be statically rendered for the index.
- Updates to the build system in general.

(1) Collecticons is very easy to remove if needed and instructions for this are provided in the README.
(2) There is no theme or base styles involved. We should decide if "normalization" styled should be a default.
(3) Didn't test other editors, but integration should be straightforward

@batpad @kamicut @vgeorge @jvntf @AliceR

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,65 +1,100 @@
...
-  build:
-    docker:
-      - image: node:8
+# ssh-keygen -m pem -t rsa -b 4096 -C developmentseed/project-seed -f deploy_key
 
-    working_directory: ~/repo
+# YAML anchors for base definitions
+base_def: &base_def
+  docker:
+    - image: node:10
```
Comment (vgeorge) [inline .circleci/config.yml:7]: Shouldn't be this node:14?

File: _README.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ The steps below will walk you through setting up your own instance of the projec
 ### Install Project Dependencies
 To set up the development environment for this website, you'll need to install the following on your system:
 
-- [Node](http://nodejs.org/) v10 (To manage multiple node versions we recommend [nvm](https://github.com/creationix/nvm))
+- [Node](http://nodejs.org/) v14 (To manage multiple node versions we recommend [nvm](https://github.com/creationix/nvm))
```
Comment (vgeorge) [inline _README.md:12]: I would suggest avoiding adding the node version to the readme as it might change and became outdated. Maybe a link to `.nvmrc`?

**Comment by vgeorge** [comment]
@danielfdsilva thanks for putting this together. I added some minor comments, overall it looks really good.

**Comment by danielfdsilva** [comment]
@vgeorge Good catches. Addressed

**Review by vgeorge — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -78,11 +106,10 @@ The script build, which uses `browserify`, outputs two js files: `bundle.js` and
    add ...`.
 
 ## Circle CI for testing and deployment
-The `.circleci/config.yml` file enables the usage of [Circle CI](http://circleci.com/) as a test and deployment system. In this particular case, Travis will be looking for any changes to the repo and when a change is made to the `master` branch, Travis will build the project and deploy it to the `gh-pages` branch.
-
+The `.circleci/config.yml` file enables the usage of [Circle CI](http://circleci.com/) as a test and deployment system. In this particular case, circle uses workflows with the linting, test, a build phases running simultaneously. When a commit is made to master the deploy phase will also run, given that the previous succeed.
 ## Linting
 
-Our [ESLint rules](.eslintrc) are based on `standard` rules, with semicolons enabled. To check  linting errors run:
+Our [ESLint rules](.eslintrc) are based on `standard` rules, with some custom options. To check linting errors run:
```
Comment (jvntf) [inline README.md:112]: looks like this has been switched to eslint recommended?

**Pull request merged**